apcd-cms/src/apps/admin_extension/views.py

```diff
@@ -44,7 +44,7 @@ def _set_extensions(extension):
                 'requested_target_date': extension[3],
                 'approved_expiration_date': extension[4],
                 'extension_type': title_case(extension[5]),
-                'applicable_data_period': extension[6].date() if extension[6] else None,
+                'applicable_data_period': datetime.datetime.strptime(str(extension[6]), '%Y%m').strftime('%b. %Y') if extension[6] else None,
```

Comment:
There's a lot of typecasting going on right here. Is there any way to simplify it? Why is all of this required for this field but not the other date fields?

Reply:
This field is the only date field that needs to be stored as an int in the form YYYYMM (e.g 202302). Not sure why that is but that's how the table is set up. I'm open to any alternative ways to do this as well